.github/workflows/o2-full-deps.yml

```diff
@@ -64,14 +64,14 @@ jobs:
           set -x
           rpmbuild -ba "rpms/o2-prereq-$DISTRO.spec"
           git clone -b "$ALIBUILD_TAG" https://github.com/alisw/alibuild
-          cd alibuild
+          cd alibuild || exit
```

Comment:
Did you check if it is running with `set -e`?

Reply:
That is a cleaner fix indeed, and actionlint doesn't seem to complain. It's now implemented